hash_practice/exercises.py

```diff
@@ -2,18 +2,47 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: 0(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍  Correct, if the words are of limited length, if the words could be any length then it would be O(n m log m)